pkgs/by-name/te/technitium-dns-server/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  dotnet-sdk_8,
+  ...
+}:
+stdenv.mkDerivation rec {
+  pname = "technitium-dns-server";
+  version = "12.0.1";
+
+  src = fetchurl {
+    url = "https://download.technitium.com/dns/archive/${version}/DnsServerPortable.tar.gz";
```

Comment:
Why not build from source?

Reply:
I tried to do that at first -- unsuccesfully. When building from source, one needs to build the dependencies as well. And then I ran into this bug: https://github.com/NixOS/nixpkgs/issues/283430. That's why I resorted to downloading the compiled version.